patches/chromium/fix_suppress_clang_-wimplicit-const-int-float-conversion_in.patch

```diff
@@ -0,0 +1,32 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: John Kleinschmidt <jkleinsc@electronjs.org>
+Date: Wed, 24 Jan 2024 11:14:50 -0500
+Subject: fix: suppress clang -Wimplicit-const-int-float-conversion in
+ v8_binding_for_core
+
+fixes the following error on build:
+
+../../third_party/blink/renderer/bindings/core/v8/v8_binding_for_core.h:272:38: error: implicit conversion from 'const uint64_t' (aka 'const unsigned long') to 'double' changes value from 18446744073709551615 to 18446744073709551616 [-Werror,-Wimplicit-const-int-float-conversion]
+  272 |   double fmod_value = fmod(trunc(d), kMaxULL + 1.0);
+      |                                      ^~~~~~~ ~
```

Comment:
The upstream code is a little odd, but what's confusing me is `v8_binding_for_core.h` hasn't changed since Thanksgiving so I'm wondering why it's just now breaking :confused: